lib/logger.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file Abtracted logging properties and methods.
+ *
+ * If for any reason, they need a different logging tool or need to be extended,
+ * the changes can be made here. 
+ */
+var util = require('util');
+var logger = require('winston');
+
+exports.logger = logger;
+
+exports.isTest = function () {
```

Comment:
@e2tha-e please add jsdoc blocks to exports. We should capture what each one is made for, and where possible, code examples on how to use these functions. [example of how to add a jsdoc example](https://github.com/pattern-library/pattern-library-utilities/blob/126c56a15dff125678f7f024409802cabeffb327/lib/gulp-tasks/gh-pages.js#L35)